DownFall/Models/EntityModel.swift

```diff
@@ -18,10 +18,18 @@ struct EntityModel: Equatable, Decodable {
         case bat
         case rat
         case dragon
+        case alamo
+        case wizard
+        case lavaHorse
         case player
+        case easyPlayer
+        case normalPlayer
+        case hardPlayer
     }
     
-    static let monsterCases: [EntityType] = [.rat, .bat, .dragon]
+    static let playerCases: [EntityType] = [.easyPlayer, .normalPlayer, .hardPlayer]
+    
+    static let monsterCases: [EntityType] = [.rat, .bat, .dragon, .alamo]
```

Comment:
This does not have all the monsters.  Also, im not sure if we use monsters cases anymore because we use Levels now

Reply:
deleted